packages/typespec-client-generator-core/test/decorators.test.ts

```diff
@@ -3306,7 +3312,8 @@ describe("typespec-client-generator-core: decorators", () => {
       strictEqual(clientAccessor.name, "getInterfaceV2");
       deepStrictEqual(clientAccessor.apiVersions, ["v2"]);
 
-      const interfaceV2 = sdkPackage.clients.find((x) => x.name === "InterfaceV2");
+      const interfaceV2 = versioningClient.methods.find((x) => x.kind === "clientaccessor")
+        ?.response as SdkClientType<SdkHttpOperation>;
```

Comment:
in the case that InterfaceV2 is a sub client of VersioningClient, how do we expect customer to initalize the client InterfaceV2 ? is it something like 
`new VersioningClient("endpoint").getInterfaceV2()` or `new VersioningClient().getInterfaceV2("endpoint")` I wonder if we need to have a way to specify where the initialization properties should belong to ?

Reply:
if the client has no public initialization, it should be initialized from parent.

Reply:
this will be a tom requirement.